modules/ROOT/pages/cors-policy.adoc

```diff
@@ -60,25 +86,51 @@ The public resource configuration looks like this:
 </http:listener-interceptors>
 ----
 
-If you cannot implement CORS for your API, another possible solution to an unreachable API is to disable the same-origin restrictions in your browser.
+== Disabling Same-Origin Restrictions in Your Browser
+
+If you cannot implement CORS for your API, you can disable the same-origin restrictions in your browser to call an inaccessible API. Each browser handles these restrictions in a unique way. You can research the tips, tricks, and plugins for your browser from the internet.
+
+Before you modify your browser settings, make sure you understand and assess its potential security implications. You must use these browser settings only for testing in your own web pages. Otherwise, the browser can become vulnerable to malicious scripts and other potential threats.
+
+=== Disabling Same-Origin Restrictions on Google Chrome for Mac OS X
+
+To disable same-origin restrictions on Google Chrome for Mac OS X:
+
+* Open a new Terminal window and paste the following line: `open -a Google\ Chrome --args --disable-web-security`.
+* Press *Enter*.
+
+=== Disabling Same-Origin Restrictions on Google Chrome for Windows
+
+To disable same-origin restrictions on Google Chrome for Windows:
+
+* Open a command prompt window.
+* Navigate to the location of the Chrome executable (Chrome.exe) file.
+* Paste the following line: `chrome.exe --disable-web-security`.
+* Press *Enter*.
 
-== Disable Same-Origin Restrictions in your Browser
+=== Disabling Same-Origin Restrictions on Internet Explorer
 
-Each browser handles these restrictions in a unique way and tips, tricks, and plugins are available on the internet.
+To disable same-origin restrictions on Internet Explorer:
 
-Make sure you understand the potential security implications of changing browser security settings. You should only use these options for testing on your own web pages because the browser can become vulnerable to malicious scripts and other potential threats.
+* Navigate to *Internet Properties* > *Security* > *Custom Level in Security Level for this Zone* > *Miscellaneous* .
+* Enable the Enable Access to Data Sources Across Domains option to access data sources across domains.
 
-=== Google Chrome for Mac OS X
+== Configuring Policy Parameters
 
-* Open a new Terminal window, paste the following line, and then press *Enter*: `open -a Google\ Chrome --args --disable-web-security`.
+When you apply the policy to your API from the UI, the following parameters are displayed:
 
-=== Google Chrome for Windows
+%header%autowidth.spread,cols="a,a,a"]
+|===
+| Element | Description | Required
+| Public resource | Whether the CORS configuration is to be applied as a public resource (default). | Yes
+| Default Group | This option is displayed when you uncheck the Public resource box. | No
+| Support credentials | Eva what are these?| No
```

Comment:
Credentials, in the context of CORS spec, are cookies, authorization headers or TLS client certificates.

Reply:
Done! Thanks!